src/components/MultiHopTrade/components/TradeSuccess/TradeSuccess.tsx

```diff
@@ -41,11 +51,17 @@ export const TradeSuccess = ({ handleBack, children }: TradeSuccessProps) => {
 
     const chainName = adapter.getDisplayName()
 
+    if (descriptionTranslation)
+      return translate(descriptionTranslation, {
+        symbol: lastHop.buyAsset.symbol,
+        chainName,
+      })
```

Comment:
preferably-blocking: this isn't generalized and is tying the translation interpolation options to `descriptionTranslation`. Can we either:
- pass `translation` from up top as a string
- pass translation args down as `titleTranslation` and `descriptionTranslation`

Reply:
Done in [3d19513](https://github.com/shapeshift/web/pull/7992/commits/3d1951386bf5e0dba41a1d9082f2a0598502f0ae)